### DISCUSSION TITLE
mc mirror stop transfer file

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#17010] [Q&A]
`c:\mc.exe mirror --watch --debug D:\share\recv myminio/tafile-backup/recv >>log\%date:~0,4%-%date:~5,2%-%date:~8,2%.log`

The transfer stops suddenly during the file transfer process, but the process is still there and no error is reported

I know the --watch will continue to listen, but the file won't be transmitted until it's fully transmitted
Resume transmission after reboot
Can it be caused by the directory being monitored being too large? Do I have any way to sense it? The process is still ongoing and there are no abnormalities. I cannot manually monitor it

mc version: [RELEASE.2022-10-29T10-09-23Z](https://github.com/minio/mc/releases/tag/RELEASE.2022-10-29T10-09-23Z)

**Comment by AlexZIX** [comment]
Hi,

few months ago I needed to sync one bucket to another on different minio clusters. The size of bucket is 10TB. mc mirror really stucks after less than 1 TB was copied and every re-run caused indexing for a hours. After 3-4 tries I've found the rclone. This tool helps me to sync buckets in a few days.
